README.md

```diff
@@ -272,17 +272,110 @@ Take a look also at [vmagent](https://docs.victoriametrics.com/vmagent.html)
 and [vmalert](https://docs.victoriametrics.com/vmalert.html),
 which can be used as faster and less resource-hungry alternative to Prometheus.
 
-## Grafana setup
+## Setup VictoriaMetrics datasource plugin for Grafana
 
-Create [Prometheus datasource](http://docs.grafana.org/features/datasources/prometheus/) in Grafana with the following url:
+VictoriaMetrics [datasource plugin](https://github.com/VictoriaMetrics/grafana-datasource) enables you to query, 
```

Comment:
It is enough just to mention that VictoriaMetrics exists and put the link. All the instructions should be a part of plugin repository, not VictoriaMetrics repo.